### PR TITLE
Set permissions for the QC config table

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: sqitch.conf [hunk 1]
```diff
@@ -1,9 +1,11 @@
 [core]
 	engine = pg
 [core "pg"]
-	db_name = genome
+	db_name = 0AAEA22466E711E5A66AA8FEA9DCD391
```
Comment (tmooney) [inline sqitch.conf]: These changes to `sqitch.conf` should probably not be included.

Comment (susannasiebert) [inline sqitch.conf]: Ooops. That wasn't intended. I amended the commit to remove the sqitch.conf changes and force pushed the new commit.

**Comment by acoffman** [comment]
+1

**Comment by tmooney** [comment]
+1

**Pull request merged**